package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@kortex-hub/mcp-registry-types",
+  "version": "0.0.1",
+  "description": "TypeScript types for the Model Context Protocol (MCP) Registry API",
+  "repository": "https://github.com/kortex-hub/kortex",
+  "publishConfig": {
+    "provenance": true,
+    "access": "public"
+  },
+  "license": "MIT",
```

Comment:
MIT? not apache 2

Reply:
the spec is in MIT so I thought here we would have to stick with the spec ?

https://github.com/modelcontextprotocol/registry/blob/main/LICENSE

Reply:
but I'm ok to change to ASL v2